proto/datastore/datastore.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package datastore;
```

Comment:
why define a new RPC? couldn't we just use gnmi.Set instead of Update()

Reply:
In the future, we'd want gnmi.Set to error out whenever the user tries to write to a read-only value. Using a different RPC call allows read-only values to be over-written.

Reply:
sorry I wasn't clear.

What i was thinking was we'd could have a local (listening unix socket) gnmi server in addition to the "real" gnmi server. (2 total). Only the local Set implementation would allow writing to read-only paths. The local server doesn't have to implement subscribe, also the Set could only implement setting to simplify server. This would allow us to allow use the easy to use ygnmi.Update/Delete.

thoughts?

Reply:
Sounds good to me, I'll make the changes.

Reply:
Done.